demo/chaincode/fpc/evaluate.cpp

```diff
@@ -215,7 +215,7 @@ void ClockAuction::DynamicAuctionState::processRegularRoundBids(
             // compute price point
             double minPrice = postedPrice_[clockRound_ - 1][eBid.territoryIndex];
             double clockPrice = clockPrice_[clockRound_][eBid.territoryIndex];
-            double pp = (eBid.demand.price_ - minPrice) / (clockPrice - minPrice);
+            double pp = (eBid.demand.price_ - minPrice) / (clockPrice - minPrice) * (100.0);
```

Comment:
It seemed to emphasize better the three blocks and separators (/*).

Reply:
Hmm, not completely following.  To emphasize the (precedence and associativity of) operators/blocks, it would seem to me  ` pp = ( (eBid.demand.price_ - minPrice) / (clockPrice - minPrice) ) * 100.0`` would be the cleanest way?  But as long as we have the same semantics of the operation, i'm fundamentally fine ... :-)